doc/nrf/known_issues.rst

```diff
@@ -184,14 +184,14 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
-.. rst-class:: v1-4-99-dev1
+.. rst-class:: v1-5-0
 
 KRKNWK-8286: Faulty UART responsiveness in Thread NCP sample
```

Comment:
This was only affecting 1.4.99. Fixed for 1.5.

Reply:
Removed the whole issue, as v1.5.0 replaces v1.4.99-dev1.